cmd/loki/Dockerfile.debug

```diff
@@ -1,8 +1,9 @@
 # Directories in this file are referenced from the root of the project not this folder
 # This file is intended to be called from the root like so:
 # docker build -t grafana/loki -f cmd/loki/Dockerfile.debug .
+ARG BUILD_IMAGE
```

Comment:
Where is the BUILD_IMAGE variable set?

Reply:
https://github.com/grafana/loki/blob/main/Makefile#L89

Reply:
I've just hardcoded the image version now as it is everywhere else too.